server/@types/licenceDates.ts

```diff
@@ -0,0 +1,10 @@
+export type LicenceDates = {
```

Comment:
There's [scripts](https://github.com/ministryofjustice/create-and-vary-a-licence#generating-types-from-openapi) to update the swagger definitions and you should be able to reference this from there? (I think)